### ISSUE TITLE
apply classification suggestions

### ISSUE THREAD

**Issue by bdionne** [NCIEVS/nci-protege5#332]
As a follow on to #320 , add the capability to apply suggestions made by the classifier

**Comment by bdionne** [comment]
[available in 0.7.3](https://github.com/bdionne/nci-edit-tab/releases/tag/nci-edit-tab-0.7.3)

**Comment by bdionne** [comment]
If suggestions appear in the classification panel, they can be automatically applied from an option in  the edit menu:

<img width="1064" alt="screen shot 2017-11-27 at 11 06 14 am" src="https://user-images.githubusercontent.com/31741/33276341-5406cdee-d363-11e7-8a1d-e40d511b4f95.png">

**Comment by bdionne** [comment]
@NCIEVS - we've modified protege now so that we can display the inferred subclasses, suitably highlighted and not editable, in the asserted view.

We should discuss, as I'm not sure this will help or further confuse modelers with respect to the suggestions in the results panel.

<img width="1024" alt="screen shot 2017-11-29 at 8 45 58 am" src="https://user-images.githubusercontent.com/31741/33378343-77cd5452-d4e2-11e7-8975-219003849427.png">

**Comment by bdionne** [comment]
We've added the explanation icon, `?`, to the inferred axioms that show up in the asserted panel. We've also added a new icon, `A`, that when clicked will convert the inferred axiom to an asserted one. It acts as a normal `edittab` edit in that the user is required to save it afterwards.

Not sure if this is better or clearer for the modelers, this would allow them to take the suggestions one at a time. Perhaps having both capabilities works well

<img width="1020" alt="screen shot 2017-11-30 at 10 13 09 am" src="https://user-images.githubusercontent.com/31741/33438014-a851eaa4-d5b7-11e7-8f3c-b466f6d6adc9.png">
